src/styles/_catalog.scss

```
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
+    display: grid;
     align-items: center;
-    width: 200px;
+    justify-content: center;
+
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 1024px) {
```

```suggestion
  @media (max-width: 1023px) {
```
There should be four columns starting at 1024px. I think you have three.